src/validation/services/offer.js

```
@@ -10,6 +16,141 @@ const allowedOfferFieldsForUpdate = {
+
+  if (existingSubject.category.toString() !== category.toString()) {
+    const err = new Error('Category does not match the subject\'s category');
+    err.status = 400;
+    throw err;
+  }
+
+  return data;
+};
+
+const validateOfferOnUpdate = async (data, offerId) => {
+  const { subject, category } = data;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)